tests/python/relay/test_op_level3.py

```diff
@@ -457,6 +457,40 @@ def test_infer_type_prelu():
     verify_infer_type_prelu((1, 3, 2, 2), None, 1, (1, 3, 2, 2))
     verify_infer_type_prelu((1, 2, 2, 3), None, 3, (1, 2, 2, 3))
 
+
+def test_arange():
```

Comment:
can we also have a relay frontend test from mxnet arange

Reply:
Sure. will add.

Reply:
Added in the new commit.